src/Plugin.tsx

```diff
@@ -38,6 +38,8 @@ const PluginInner = (propsFromParent: EnrollmentOverviewProps) => {
         isWeightInGrams: chartConfig?.settings.weightInGrams || false,
     });
 
+    const isPercentiles = chartConfig?.settings.defaultStandard === 'p';
```

Comment:
I think it would be a good idea to operate with only true and false. Change the prop name to `percentiles` etc. in **config file**, and update implementation guide.